src/App.js

```diff
@@ -10,10 +10,32 @@ const tabs = [
 ];
 /* eslint-enable */
 
-const App = () => (
-  <div className="App">
-    <h1>Selected tab is&nbsp;Tab 1</h1>
-  </div>
-);
+class App extends React.Component {
+  state = {
+    tabs: [...tabs],
+    selectedTab: 'tab-1',
```

Comment:
краще видягувати з масиву табу, яка має бути активною
```suggestion
    selectedTab: tabs[0].id,
```